src/util/string_converter.cc

```diff
@@ -92,7 +92,7 @@ std::string StringConverter::_convert(const std::string& str, bool validate,
 
     auto input = str.c_str();
     size_t length = str.length();
-    if (length < (std::numeric_limits<size_t>::max() / 4)) {
+    if (length < (std::string::npos / 4)) {
```

Comment:
From the definition shown in https://en.cppreference.com/w/cpp/string/basic_string/npos this may cause overflows between signed and unsigned

Reply:
How so? npos is size_type which is the same as size_t. The -1 evaluates to max()

Reply:
Then I'm a bit too much afraid of integer overflow. :smile: